sufia.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'select2-rails', '~> 3.5.9'
   spec.add_dependency 'json-schema'
   spec.add_dependency 'oauth'
+  spec.add_dependency 'devise', '~> 4.2'
```

Comment:
There is a deprecation warning from spec_helper in certain versions of devise.  The warning references an alternative invocation that I used, but the alternative invocation is not available on *all* versions of devise.  This specification allows the deprecation to be avoided, but also not crash the build.